recipe/build.sh

```diff
@@ -5,12 +5,14 @@ set -ex
 mkdir -pv build
 pushd build
 
+
 # replace freedstock name in debug-prefix-map with source name
 export CFLAGS=$(echo ${CFLAGS:-} | sed -E 's|'\/usr\/local\/src\/conda\/${PKG_NAME}'|/usr/local/src/conda/Fr|g')
 
 # configure
 cmake ${SRC_DIR} ${CMAKE_ARGS} \
 	-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo \
+        -DCMAKE_C_COMPILER=$CC \
```

Comment:
There should be no need to set this explicitly. `cmake` should pick that up from environment variables and `CMAKE_ARGS`.

Reply:
@erinelizabethseder-virgo, can you please revert this change, or better yet, just `drop` the commits that modify this file.